pytorch_lightning/__init__.py

```diff
@@ -35,7 +35,7 @@
 
 import logging as python_logging
 
-_logger = python_logging.getLogger("lightning")
+_logger = python_logging.getLogger(__name__)
```

Comment:
@awaelchli  Could you explain why this fixes the DDP issue? I don't quite follow

Reply:
@ananthsub so the duplicate logging is not ddp related. in fact, I observed this on single gpu, and even cpu. as I said only on windows, but I suspect there is just a subtle difference between how these logging imports work between python versions and platforms. For some reason, if we don't use the module name as name for the logging instance, it creates two of them who independently log. as I said, found his fix by accident, I do not have an explanation, only empirical evidence that it fixes it on my machines.